README.md

```diff
@@ -3,9 +3,17 @@
 ## Usage
 
 ```
-$ npx pokefusion
+$ npx pokefusion [pokemon1] [pokemon2]
 ```
 
+Pokemon can be specified as names or pokedex numbers:
+```
+pokefusion charizard bulbasaur
+pokefusion 6 1
```

Comment:
Can we pop an example in here for Nidoran F/M or Farfetch'd so people know how to handle Pokemon with spaces or non alpha chars.